src/cargo/util/command_prelude.rs

```diff
@@ -308,7 +308,8 @@ pub trait ArgMatchesExt {
                 if self._is_present(flag) {
                     bail!(
                         "--{} is not allowed in the root of a virtual workspace\n\
-                         note: while this was previously accepted, it didn't actually do anything",
+                         note: while this was previously accepted, it didn't actually do anything\n\
+                         help: try cd into the package and run cargo command",
```

Comment:
It might be good to also mention `--manifest-path`.  Maybe something like this:

> help: change the current directory to the package directory, or use the `--manifest-path` flag to the path of the package

Reply:
Ah, I didn't know about the manifest path, I thought it will automatically find it?